src/libstd/primitive_docs.rs

```diff
@@ -362,8 +362,13 @@ mod prim_unit { }
 ///
 /// *[See also the `std::ptr` module](ptr/index.html).*
 ///
-/// Working with raw pointers in Rust is uncommon,
-/// typically limited to a few patterns.
+/// Working with raw pointers in Rust is uncommon, typically limited to a few patterns.
+/// Raw pointers can be unaligned or [`null`] when unused. However, when a raw pointer is
```

Comment:
```suggestion
/// Raw pointers can be unaligned or [`null`]. However, when a raw pointer is
```

You can use `read_unaligned` to read from a raw pointer that's not unaligned - I'd say that this is a "use" of the raw pointer, and that therefore raw pointers can be unaligned when used as long as they are used appropriately. So I'd just remove the "when unused" part here.